src/main/webapp/i18n/en/editor.json

```diff
@@ -6,17 +6,26 @@
             },
             "title": "Code Editor for Exercise:",
             "instructions": "&nbsp; Instructions &nbsp;",
-            "unsavedChanges": "There are unsaved changes.",
+            "unsavedChanges": "Unsaved.",
+            "unsavedChangesTooltip": "There are unsaved changes.",
             "savingChanges": "Saving changes.",
+            "savingChangesTooltip": "Saving...",
```

Comment:
Shouldn't the text for savingChanges and savingChangesTooltip be switched? :)

Reply:
Correct :)